simple_https_client/src/lib.rs

```diff
@@ -74,11 +83,23 @@ where
     // TODO: Who should do this check?
     if eq1(server_hello[0], U8(21)) {
         // Alert
-        error!("Server does not support proposed algorithms.");
+        eprintln!("Server does not support proposed algorithms.");
```

Comment:
Better use logging?

Reply:
Logging is noisy and can be disabled. This needs to always go out.